bpdm-pool-api/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/api/model/request/LegalEntityPropertiesSearchRequest.kt

```diff
@@ -21,14 +21,30 @@ package org.eclipse.tractusx.bpdm.pool.api.model.request
 
 import io.swagger.v3.oas.annotations.Parameter
 import io.swagger.v3.oas.annotations.media.Schema
-
+import org.eclipse.tractusx.bpdm.gate.api.model.StreetDto
 
 @Schema(description = "Contains keywords used for searching in legal entity properties")
 data class LegalEntityPropertiesSearchRequest(
     @field:Parameter(description = "Filter legal entities by name")
-    val legalName: String?
+    val legalName: String?,
+
+    @field:Parameter(description = "Filter business partners by CX-BPN.")
+    val bpn: String?,
+
+    @field:Parameter(description = "Filter business partners by street name.")
+    val street: StreetDto?,
```

Comment:
Why is the whole StreetDto here? I can only see the street name field being used for the search. I recommend changing this field then to a string and rename it to streetname for clarity